x-pack/test/functional/apps/maps/group4/visualize_create_menu.js

```diff
@@ -16,9 +16,12 @@ export default function ({ getService, getPageObjects }) {
     describe('maps visualize alias', () => {
       describe('with write permission', () => {
         before(async () => {
-          await security.testUser.setRoles(['global_maps_all', 'global_visualize_all'], {
-            skipBrowserRefresh: true,
-          });
+          await security.testUser.setRoles(
```

Comment:
ℹ️ I am adding another role to this user here. The `test_logstash_reader` adds permission to this user to the logstash index. The new no data component for visualize needs ES data and dataviews to exist.